app/views/book/show.html.erb

```diff
@@ -6,16 +6,25 @@
   <%= render partial: 'book/header' %>
 
   <% @book.next_lesson_for(current_user).try do |lesson| %>
-    <div class="actions">
-      <a href="<%= lesson_path(lesson) %>" class="btn btn-complementary">
-        <%= t book_practice_key_for(current_user) %>
-      </a>
-    </div>
+    <% if show_content_element? %>
+      <div class="actions">
+        <a href="<%=   lesson_path(lesson)   %>" class="btn btn-complementary">
+          <%=   t book_practice_key_for(current_user)   %>
+        </a>
+      </div>
+    <%  else  %>
+      <br>
+    <%   end %>
```

Comment:
just being nitpicky but all the double/triple spaces in this file are triggering me